app/models/fixed_asset.rb

```diff
@@ -193,6 +193,11 @@ class FixedAsset < Ekylibre::Record::Base
     depreciate! if @auto_depreciate
   end
 
+  def any_financial?
+    self.started_on <= self.journal.closed_on ? false : true
```

Comment:
The return value must match the name. If the name is `any_financial?` or `on_closed_periods?` it should return true when it is on closed periods or when there are financial_years closed on the duration. Right now the return value is the opposite of what would be expected by reading the name.
Either the name of the method needs to change to match the behavior, or the behavior must change to match the method name. (Note: it is possible to use `transition ... unless: <method>'  just like you used the `transition ... if: <method>` in the state_machine declaration.)